modules/vaos/app/controllers/vaos/v2/scheduling_controller.rb

```diff
@@ -4,23 +4,20 @@ module VAOS
   module V2
     class SchedulingController < VAOS::V0::BaseController
       def configurations
-        facility_ids = configuration_params[:facility_ids]
-        cc_enabled = configuration_params[:cc_enabled]
-
-        response = mobile_facility_service.get_scheduling_configurations(facility_ids, cc_enabled)
+        response = mobile_facility_service.get_scheduling_configurations(csv_facility_ids, params[:cc_enabled])
         render json: VAOS::V2::SchedulingConfigurationSerializer.new(response[:data], meta: response[:meta])
       end
 
       private
 
-      def configuration_params
-        params.require(:facility_ids)
-        params.permit(:cc_enabled, facility_ids: [])
-      end
-
       def mobile_facility_service
         VAOS::V2::MobileFacilityService.new(current_user)
       end
+
+      def csv_facility_ids
```

Comment:
csv? when did we start working with files instead of JSON? 🤔

Reply:
these are for the query parameters we send to VAMF Mobile Facilities Service. The Mobile Facility Service requires the list to be comma separated values. We still work with JSON but in the request and response bodies.

Reply:
ah, my bad I thought it was a request body. This makes sense... i was thinking the faraday middleware handles those, but it does not.

Reply:
k, go ahead and merge this -- but we've ran into this problem before and used FlatParamsEncoder with faraday to convert.